src/utils/redux.js

```diff
@@ -0,0 +1,25 @@
+import { combineReducers } from 'redux';
+
+import React from 'react';
+
+import createVmWizardReducers from '../components/Wizard/CreateVmWizard/redux/reducers';
+
+let lastId = 0;
+
+export const getNextReduxId = () => ++lastId;
+
+export const withReduxId = Component => {
```

Comment:
id is created for each new wizard - this could be used for remembering values or opening multiple dialogs at once